img/logos/salalem.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Converted their logo into a SVG since it was missing.

Reply:
Thanks!